### PR TITLE
update to 0.4.6

### PR DESCRIPTION
**Pull request by boldorider4**
* [x] checked license
* [x] updated dependencies with [upstream](https://github.com/pythological/unification/blob/main/requirements.txt)
* [x] updated about section

### CODE REVIEW

**Review by varlackc — APPROVED** [review]
LGTM

**Review by M-Waszkiewicz-Anaconda — APPROVED** [review]
LGTM

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,39 +1,47 @@
...
-    - python >=3.6
+    - python
     - setuptools
+    - wheel
   run:
-    - python >=3.6
-    - toolz
-    - multipledispatch
+    - python
+    - versioneer
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]: `toolz` and `multipledispatch` are real dependencies https://github.com/pythological/unification/blob/94e489e6cf583f24a9af2312b2eefd8aa4949a4d/setup.py#L19-L21
`versioneer` is a **dev** dependency

Comment (boldorider4) [reply recipe/meta.yaml]: Thanks for the catch! You're right, this was a mere mistake.

**Review by skupr-anaconda — APPROVED** [review]
LGTM

**Pull request merged**